ChangeLog

```diff
@@ -28,6 +28,7 @@ English Dolibarr ChangeLog
 - Fix: [ bug #1825 ] External agenda: hide/show checkbox doesn't work
 - Fix: [ bug #1790 ] Email form behaves in an unexpected way when pressing Enter key
 - Fix: Bad SEPA xml file creation
+- Fix: [ bug #1856 ] ODT Templates - Templating keys are not always recognized
```

Comment:
You're not supposed to update the ChangeLog anymore. Please see: https://github.com/Dolibarr/dolibarr/blob/develop/CONTRIBUTING.md#general-rules